script/DeploySystem.s.sol

```diff
@@ -229,6 +229,12 @@ contract DeploySystem is Deploy {
 
         bytes32 batcherHash = bytes32(uint256(uint160(cfg.batchSenderAddress())));
 
+        address customGasTokenAddress = Constants.ETHER;
+        if (cfg.useCustomGasToken()) {
+            customGasTokenAddress = cfg.customGasTokenAddress();
+            console.log("configured to use custom gas token %s", customGasTokenAddress);
+        }
+
```

Comment:
We shouldn't be updating `DeploySystem.s.sol`, as this is only run once. We should instead modify `DeployChain.sol` so that each individual deploy can have a different gas token.

The contracts / proxies deployed by `DeploySystem` aren't actually used by any by the deployed chains.

Reply:
Got it, will update! Thanks!